plugins/modules/win_domain_computer.ps1

```diff
@@ -20,9 +23,9 @@ $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "b
 $diff_support = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -default $false
 
 $name = Get-AnsibleParam -obj $params -name "name" -failifempty $true -resultobj $result
-$sam_account_name = Get-AnsibleParam -obj $params -name "sam_account_name" -default "$name$"
+$sam_account_name = Get-AnsibleParam -obj $params -name "sam_account_name" -default "${name}$"
```

Comment:
Curious is the `${name}` change just a style change or does this actually change the behaviour. I would have thought the original code was fine?

Reply:
Style only; a little clearer imo. I can revert but in this case I don't think it matters much git blame wise because of the collection split.

Reply:
That's ok, it's a benign change and I was just curious if it was done for a particular reason.